src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -179,11 +190,23 @@ codegenExpr codegenEnv@{ currentModule } s = case unwrap s of
     S.runUncurriedFn
       (S.Identifier $ rtPrefixed "array-ref")
       [ codegenExpr codegenEnv e, S.Integer $ wrap $ show i ]
+  Accessor e
+    (GetCtorField (Qualified (Just (ModuleName "Data.List.Types")) (Ident "Cons")) _ _ _ "value0" _) ->
+    S.List [ S.Identifier (scmPrefixed "car"), (codegenExpr codegenEnv e) ]
+  Accessor e
+    (GetCtorField (Qualified (Just (ModuleName "Data.List.Types")) (Ident "Cons")) _ _ _ "value1" _) ->
+    S.List [ S.Identifier (scmPrefixed "cdr"), (codegenExpr codegenEnv e) ]
   Accessor e (GetCtorField qi _ _ _ field _) ->
     S.recordAccessor (codegenExpr codegenEnv e) (flattenQualified currentModule qi) field
   Update e f ->
     S.recordUpdate (codegenExpr codegenEnv e) (map (codegenExpr codegenEnv) <$> f)
 
+  CtorSaturated (Qualified (Just (ModuleName "Data.List.Types")) (Ident "Nil")) _ _ _ _ ->
+    S.List [ S.Identifier $ scmPrefixed "quote", S.List [] ]
```

Comment:
Thoughts on using a symbol here? If we ever need to unsafely print some scheme code, `quote ()` may not be that helpful.

Reply:
You mean that we could instead use `(list)` or something else? I thought this was just a scheme-y way of doing this. We also use `quote` elsewhere in codegen.

Reply:
Or do you mean using `'()`?

Reply:
I'm not very familiar with Scheme, and it's been some time since I was learning its syntax. Is `quote ()` what Schemers typically use to indicate the end of singly-linked lists?
Initially, I was thinking we should have a value that differs from the Scheme list to help us know when this value is from Scheme code (e.g. FFI) and when it's from PureScript code. However, if we're using native lists, then I guess that's dumb. We'd want to use what Scheme users would use.

Reply:
Yes, `'()` (where `'` is equivalent to `quote`) is the idiomatic way to express the end of the list.

From the repl:
```scheme
> (cons 'a (cons 'b (cons 'c '())))
(a b c)

```

Reply:
Thanks for clarifying!